vampire_app/app.js

```diff
@@ -14,29 +18,306 @@
+// 			eye_color: "brown",
+// 			dob: new Date(1990,6,22,3,0),
+// 			loves: ["Mia","THC"],
+// 			location: "Somerville, Massachusetts, US",
+// 			gender: "m",
+// 			title: "Konvurt",
+// 			victims: 7
+// 		}, (err, createdVampire) => {
+// 			if(err) {
+// 				console.log(err);
+// 			} else {
+// 				console.log(createdVampire);
+// 				Vampire.create({
+// 					name: "Mary",
+// 					hair_color: "blonde",
+// 					eye_color: "blue",
+// 					dob: new Date(1992,1,24,0,0),
+// 					loves: ["fuzzy animals", "gifts"],
+// 					location: "New York, New York, US",
+// 					gender: "f",
+// 					title: "Marmaduke",
+// 					victims: 50
+// 				}, (err, createdVampire) => {
+// 					if(err) {
+// 						console.log(err);
+// 					} else {
+// 						console.log(createdVampire);
+// 						Vampire.create({
+// 							name: "Kayla",
+// 							hair_color: "black",
+// 							eye_color: "brown",
+// 							dob: new Date(1989,9,15,3,0),
+// 							loves: ["Seth","nerd stuff"],
+// 							location: "Somerville, Massachusetts, US",
+// 							gender: "f",
+// 							title: "Kalyco",
+// 							victims: 1
+// 						}, (err, createdVampire) => {
+// 							if(err) {
+// 								console.log(err);
+// 							} else {
+// 								console.log(createdVampire);
+// 							}
+// 						})
+// 					}
+// 				})
+// 			}
+// 		})
+// 	}
+// })
 
 /////////////////////////////////////////////////
 // ## QUERYING
 /////////////////////////////////////////////////
 // ### Select by comparison
+// Vampire.find({gender: 'f'}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
+
+// Vampire.find({victims: {$gt: 500}}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
+
+// Vampire.find({victims: {$lte: 150}}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
+
+// Vampire.find({victims: {$ne: 21034}}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
+
+// Vampire.find({victims: {$gt: 150, $lt: 500}}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
 
 /////////////////////////////////////////////////
 // ### Select by exists or does not exist
+// Vampire.find({title: {$exists: true}}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
+
+// Vampire.find({victims: {$exists: true, $gt: 1000}}, (err, foundVamps) => {
+// 	if(err) {
+// 		console.log(err);
+// 	} else {
+// 		console.log(foundVamps);//will return matches as an array
+// 	}
+// });
 
```

Comment:
what about: the following prompts?
have a title AND no victims
have victims AND the victims they have are greater than 1000